seed/supabase-twitter-clone/README.md

```diff
@@ -71,23 +70,11 @@ First, let's set up a local development environment for the Supabase Twitter clo
     # Your projectID can be found using the `supabase projects list` command and noting the REFERENCE ID value.
     # Input your remote database password when prompted.
     npx supabase link --project-ref <your-twitter-clone-project-id>
-    # Create a valid migrations folder for Supabase to pull the first migration.
-    mkdir -p supabase/migrations
+
     # Pull the database schema from the remote project.
     npx supabase db pull
     ```
 
-This process creates a new `remote_schema.sql` file within the `supabase/migrations` folder. However, this migration lacks the necessary triggers and publications for our real-time updates to function correctly. Thus, we need to manually add them to the `remote_schema.sql` file:
-
-```sql
--- Append at the end
--- Trigger to create a profile for a user upon creation
-CREATE TRIGGER on_auth_user_created AFTER INSERT ON auth.users FOR EACH ROW EXECUTE FUNCTION "public"."create_profile_for_user"();
--- Publication for the tweets table to enable real-time functionality
-ALTER PUBLICATION "supabase_realtime" ADD TABLE "public"."tweets";
-RESET ALL;
-```
```

Comment:
These were already in the migration file that was checked into the repo, so I guessed we don't need these lines in the readme anymore: https://github.com/snaplet/examples/blob/main/seed/supabase-twitter-clone/supabase/migrations/20240312180754_remote_schema.sql#L169-L171